package.json

```diff
@@ -100,7 +94,11 @@
     "wdio-locale": "LOCALE=de wdio",
     "wdio-lowlight": "THEME=clinical-lowlight-theme wdio",
     "wdio-fusion": "THEME=orion-fusion-theme wdio",
-    "wdio": "npm run wdio-default && npm run wdio-locale && npm run wdio-lowlight && npm run wdio-fusion"
+    "wdio": "npm run wdio-default && npm run wdio-locale && npm run wdio-lowlight && npm run wdio-fusion",
+    "version:major": "npm --no-git-tag-version version major",
+    "version:minor": "npm --no-git-tag-version version minor",
+    "version:patch": "npm --no-git-tag-version version patch",
+    "postversion": "node ./scripts/changelog-updater/index.js"
```

Comment:
Is this how npm works?  postversion does post of version:<blah>?

Reply:
postversion does a post of version, i added the major minor patch scripts to make it easier to not use git tagging

Reply:
Ideally i'd like to align this with the UI that lerna has to select a version, but this is the quick way to get our releases aligned.

Reply:
postversion is documented here https://docs.npmjs.com/cli/version

Reply:
> Ideally i'd like to align this with the UI that lerna has to select a version, but this is the quick way to get our releases aligned.

Could you link the lerna UI? I'm curious about our own version scripts instead of using lerna's

Reply:
this is the UI i'm referencing: 
![Screen Shot 2020-09-03 at 12 16 56 PM](https://user-images.githubusercontent.com/1588747/92146522-7aaefe00-eddf-11ea-9c9a-45a5c8c85bdd.png)